server/backend/feedback.go

```diff
@@ -0,0 +1,142 @@
+package backend
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"path"
+
+	pb "github.com/TUM-Dev/Campus-Backend/server/api/tumdev"
+	"github.com/TUM-Dev/Campus-Backend/server/backend/cron"
+	"github.com/TUM-Dev/Campus-Backend/server/model"
+	"github.com/gabriel-vasile/mimetype"
+	"github.com/gofrs/uuid/v5"
```

Comment:
Could we use something more supported here?
e.g.: https://github.com/google/uuid

Reply:
We can, but in another PR
(The reason I switched to `gofrs/uuid` was, that this is the successor of the previously used library and does not have the security flaws of the original library [big-doubt, that we could be pawned, but less than random randomess is still bad])